srcpkgs/cmake-bootstrap/template

```diff
@@ -0,0 +1,52 @@
+# Template file for 'cmake-bootstrap'
+pkgname=cmake-bootstrap
+version=3.18.2
+revision=1
+wrksrc=cmake-$version
+bootstrap=yes
+build_style=configure
+configure_args="--prefix=/usr --datadir=share/$pkgname --bindir=bin
+ --system-libs --no-system-curl --no-system-jsoncpp --no-system-nghttp2
+ --no-system-librhash --no-system-libuv --no-system-expat --no-qt-gui
+ ${XBPS_MAKEJOBS:+--parallel=$XBPS_MAKEJOBS}"
+make_build_args="-C Bootstrap.cmk"
+makedepends="libarchive-devel"
+short_desc="CMake (bootstrapping version)"
+maintainer="Đoàn Trần Công Danh <congdanhqx@gmail.com>"
+license="LGPL-2.1-or-later, BSD-3-Clause"
+homepage="https://www.cmake.org"
+distfiles="https://www.cmake.org/files/v${version%.*}/cmake-${version}.tar.gz"
+checksum=5d4e40fc775d3d828c72e5c45906b4d9b59003c9433ff1b36a1cb552bbd51d7e
+conflicts="cmake>=0"
+
+CFLAGS="-DCMAKE_USE_SYSTEM_ZLIB -DCMAKE_USE_SYSTEM_LIBARCHIVE"
+CXXFLAGS="$CFLAGS"
+
+post_patch() {
+	rm -rf Utilities/cmbzip2
+	rm -rf Utilities/cmcurl
+	rm -rf Utilities/cmexpat
+	# rm -rf Utilities/cmjsoncpp
```

Comment:
why is this commented?

Reply:
Originally, I wanted to remove all 3rd party libraries except `libuv` since I know that `libuv` is required. I found out `jsoncpp` was required later on.